src/wd.rs

```diff
@@ -373,3 +372,440 @@ impl NewSessionResponse {
+
+        self
+    }
+}
+
+/// The print configuration.
+#[derive(Debug, Clone, PartialEq)]
+pub struct PrintConfiguration {
+    orientation: PrintOrientation,
+    scale: f64,
+    background: bool,
+    size: PrintSize,
+    margins: PrintMargins,
+    page_ranges: Vec<PrintPageRange>,
+    shrink_to_fit: bool,
+}
+
+impl PrintConfiguration {
+    /// Creates a [`PrintConfigurationBuilder`] to configure a [`PrintConfiguration`].
+    pub fn builder() -> PrintConfigurationBuilder {
+        PrintConfigurationBuilder::default()
+    }
+
+    pub(crate) fn into_params(self) -> PrintParameters {
+        PrintParameters {
+            orientation: self.orientation.into_params(),
+            scale: self.scale,
+            background: self.background,
+            page: self.size.into_params(),
+            margin: self.margins.into_params(),
+            page_ranges: self
+                .page_ranges
+                .into_iter()
+                .map(|page_range| page_range.into_params())
+                .collect(),
+            shrink_to_fit: self.shrink_to_fit,
+        }
+    }
+}
+
+impl Default for PrintConfiguration {
+    fn default() -> Self {
+        PrintConfiguration {
+            orientation: PrintOrientation::default(),
+            scale: 1.0,
+            background: false,
+            size: PrintSize::default(),
+            margins: PrintMargins::default(),
+            page_ranges: Vec::new(),
+            shrink_to_fit: true,
+        }
+    }
+}
+
+/// The orientation of the print.
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Default)]
+pub enum PrintOrientation {
+    /// Landscape orientation.
+    Landscape,
+    #[default]
+    /// Portrait orientation.
+    Portrait,
+}
+
+impl PrintOrientation {
+    pub(crate) fn into_params(self) -> webdriver::command::PrintOrientation {
+        match self {
+            Self::Landscape => webdriver::command::PrintOrientation::Landscape,
+            Self::Portrait => webdriver::command::PrintOrientation::Portrait,
+        }
+    }
+}
+
+/// The size of the printed page in centimeters.
+#[derive(Debug, Clone, PartialEq)]
+pub struct PrintSize {
+    /// The width in centimeters.
+    pub width: f64,
+    /// The height in centimeters.
+    pub height: f64,
+}
+
+impl PrintSize {
+    /// The standard A4 paper size, which has the dimension of `21.0x29.7cm`.
+    pub const A4: Self = Self {
+        width: 29.7,
+        height: 21.0,
+    };
+
+    /// The standard US letter paper size, which has the dimension of `21.59x27.94cm`.
+    pub const US_LETTER: Self = Self {
+        width: 21.59,
+        height: 27.94,
+    };
+
+    /// The standard US legal paper size, which has the dimension of `21.59x35.56cm`.
+    pub const US_LEGAL: Self = Self {
+        width: 21.59,
+        height: 35.56,
+    };
+
+    /// The minimum page size allowed by the Webdriver2 standard, which is `2.54/72cm`.
+    pub const MIN: Self = Self {
+        // FIXME: use 2.54/72.0 with MSRV >= 1.82
```

Comment:
Why is the MSRV relevant here?

Reply:
The MIN value (0.036) is an approximate result of the 2.54/72.0 operation.
Without the approximation, the result would be similar to 0.03527777777777778, but unfortunately constant floating point operations were introduced in [Rust 1.82](https://blog.rust-lang.org/2024/10/17/Rust-1.82.0.html#floating-point-nan-semantics-and-const).

Reply:
Ohhh, okay, yeah, makes sense, thanks!